src/main/resources/explanations/challenge41.adoc

```diff
@@ -0,0 +1,9 @@
+=== Password Shucking
+
+A website was using https://en.wikipedia.org/wiki/MD5[MD5] for hashing passwords, and its developers recently decided that it's a good idea to migrate to a stronger hashing algorithm like https://en.wikipedia.org/wiki/Bcrypt[bcrypt].
+
+The developers decided that the easiest way to migrate would be to hash the pre-existing hashes using bcrypt. Using two hashing algorithms would be more secure than using one, right? It appears so.
+
+For this challenge, you are provided with two database dumps containing usernames and passwords. The dump file https://github.com/OWASP/wrongsecrets/tree/master/src/main/resources/executables/db-dump.txt[db-dump.txt] was generated before the migration and the other dump file https://github.com/OWASP/wrongsecrets/tree/master/src/main/resources/executables/db-dump-2.txt[db-dump-2.txt] was generated after the migration. Both dump files are available inside the https://github.com/OWASP/wrongsecrets/tree/master/src/main/resources/executables[executables folder].
```

Comment:
Can we maybe add that the devs got into the unfortunate sitation that their db got dumped, hence they wanted to upgrade and then it got dumped again in a second hack? That way it looks more like a scenario :) .